its/ruling/src/test/resources/expected/python-S5706.json

```diff
@@ -0,0 +1,6 @@
+{
+'project:S5706.py':[
```

Comment:
We usually don't do that anymore. I guess it was a good exercise in this case :)